src/api-service/__app__/queue_file_changes/__init__.py

```diff
@@ -29,10 +29,10 @@ def file_added(event: Dict, fail_task_on_transient_error: bool) -> None:
 def main(msg: func.QueueMessage, dashboard: func.Out[str]) -> None:
     event = json.loads(msg.get_body())
     last_try = msg.dequeue_count == MAX_DEQUEUE_COUNT
-    if event["topic"] not in corpus_accounts():
+    if event["eventType"] != "Microsoft.Storage.BlobCreated":
```

Comment:
Could you add a short comment emphasizing why this order is preferable?

Reply:
```suggestion
    # check type first before calling Azure APIs
    if event["eventType"] != "Microsoft.Storage.BlobCreated":
```